app/pages/preview/ui-tests/withSavedData.ui.test.js

```diff
@@ -89,6 +89,21 @@ const mockOrder = {
       deliveryDate: '2020-09-25',
       costPerYear: 29940.00,
     },
+    {
+      itemId: 'C000001-01-A10001-4',
+      serviceRecipientsOdsCode: 'A10001',
+      cataloguePriceType: 'Flat',
+      catalogueItemType: 'AdditionalService',
+      catalogueItemName: 'Remote Consultation',
+      provisioningType: 'Declarative',
+      price: 207.916,
+      itemUnitDescription: 'per practice',
+      timeUnitDescription: 'per month',
+      quantityPeriodDescription: 'per year',
+      quantity: 12,
+      deliveryDate: '2020-09-25',
+      costPerYear: 29940.00,
```

Comment:
shouldn't the cost be 29,939.90?
I realise the other tests may have this wrong too.